lib/ts/recipe/multitenancy/utils.ts

```diff
@@ -90,11 +90,12 @@ export const mergeProviders = ({
     const providers: Pick<Provider, "id" | "buttonComponent" | "getButton">[] = [];
 
     for (const tenantProvider of tenantProviders) {
-        // try finding exact match first
+        // try finding exact match or client provider that includes tenant id
         let provider = clientProviders.find((provider) => {
             const { id } = tenantProvider;
             return provider.id === id || provider.id.includes(id);
```

Comment:
This `includes` is a bit strange to me here. Could you point me to the docs about this merging algorithm?

Reply:
i can remove this include, i dont remember why i added this